docs/source/config/details.rst

```diff
@@ -34,7 +34,7 @@ which defines the defaults. The required interface is like this:
    :class:`~.TerminalInteractiveShell` instance.
 
    .. method:: in_prompt_tokens(cli=None)
-               continuation_prompt_tokens(self, cli=None, width=None)
+               continuation_prompt_tokens(self, width=None)
```

Comment:
While you are at it, maybe remove `cli=None` from the previous method, and the two lines 46-47 below.